### PR TITLE
Scikit-learn required for plot_param_importances.

### PR DESCRIPTION
**Pull request by ll7**
## Motivation
plot_param_importances requires scikit-learn and this requirement is not mentioned

## Description of the changes
mention the requirement in the documentation

### CODE REVIEW

**Review by himkt — APPROVED** [review]
LGTM. The current evaluators depend on scikit-learn.

- https://optuna.readthedocs.io/en/stable/reference/generated/optuna.importance.FanovaImportanceEvaluator.html#optuna.importance.FanovaImportanceEvaluator
- https://optuna.readthedocs.io/en/stable/reference/generated/optuna.importance.MeanDecreaseImpurityImportanceEvaluator.html#

File: docs/source/reference/visualization/index.rst [hunk 1]
```diff
@@ -8,7 +8,7 @@ The :mod:`~optuna.visualization` module provides utility functions for plotting
 .. note::
     In the :mod:`optuna.visualization` module, the following functions use plotly to create figures, but `JupyterLab`_ cannot
     render them by default. Please follow this `installation guide`_ to show figures in
-    `JupyterLab`_.
+    `JupyterLab`_. The `plot_param_importances` requires the pip package `scikit-learn`.
```
Comment (toshihikoyanase) [inline docs/source/reference/visualization/index.rst:11]: This note block originally focused on the use of the visualization functions in `JupyterLab`, but the `scikit-learn` package is required even if users do not use `JupyterLab`. 

image

How about creating a new note block, for example, as follows?

```suggestion
    `JupyterLab`_. 
.. note::
    The :func:`~optuna.visualization.plot_param_importances` requires the Python package of `scikit-learn <https://github.com/scikit-learn/scikit-learn>`_.
```

**Review by toshihikoyanase — COMMENTED** [review]
Thank you for your PR! I added a small comment.

File: docs/source/reference/visualization/index.rst (hunk 1 above)
Comment (toshihikoyanase) [reply docs/source/reference/visualization/index.rst:11]: @ll7 If you are busy, I'll commit the suggestion to your branch. @himkt will review the change in that case, so please feel free to ask us.

**Comment by toshihikoyanase** [comment]
@himkt I committed my suggestion to the branch. Could you check it, please?
https://github.com/optuna/optuna/pull/3332/commits/358b72c983575e3eed3b1d5bf2fb001b47b60d17

**Review by himkt — APPROVED** [review]
LGTM. Thank you @ll7 for your contribution!

**Comment by himkt** [comment]
Cov check also passes. 🚀

**Pull request merged**